std/bigint.d

```diff
@@ -1025,6 +1025,193 @@ public:
         assert(aa[BigInt(456)] == "def");
     }
 
+    /*
+        Increments an arbitrary-length integer, represented as an array of
+        bytes, using the native endianness.
+
+        This is just used by `toBytes` to convert unsigned bytes of the
+        BigInt to two's complement form.
+    */
+    private static void incrementBytes (ref ubyte[] value) nothrow pure @safe
+    {
+        if (value.length == 0u)
+        {
+            value = [ 0x01u ];
+            return;
+        }
+
+        version (BigEndian)
+        {
+            // This loop adds one to an arbitrary length array of bytes.
+            for (size_t i = value.length - 1; i < 0u; i--)
```

Comment:
This would be `foreach_reverse (0u ... value.length - 1)` or `foreach_reverse (e; size_t(0) .. 10)` if you want `size_t` - though I think in this case even `int` is fine.
Also note that `i < 0` can never be true and that IIRC `BigEndian` isn't tested on any CIs.